src/pages/user/UserRoles.tsx

```diff
@@ -21,7 +21,7 @@ const UserRoles: FC<UserRolesProps> = ({ user }) => {
   return (
     <Card>
       <Typography variant="h5">{t('heading.roles')}</Typography>
-      {user.customerId ? (
+      {user.customerId && !user.customerId?.endsWith('/customer/None') ? (
```

Comment:
snakket med Jan om dette? jeg syns det høres ut som en backend-bug. trodde vi enten skal få en gyldig URI eller ingenting. tror vi har en enkel sjekk på `customerId` flere andre plasser også, så det blir evt en større fiks :/

Reply:
Hmm, tenker at det er bedre at vi får noe som vi faktisk kan identifisere som noen som ikke er customer enn bare et tomt felt.

Reply:
Ser at vi sjekker for tom verdi  i `Routes.tsx` også

Reply:
jeg ville foretrukket `customerId: null` eller `customerId: ""` fra backend. Gir ingen verdi å få `customerId: "/customer/None"` for oss...

Reply:
OK, skal snakke med Jan da

Reply:
Jan sier at for tiden tillates det ikke tomme strenger, men at de skal se på det når de får en egen modell for Dto'er

Reply:
Legger denne på is foreløpig da

Reply:
hvis det vil ta litt tid før backend får fikset det kan vi evt legge inn en quick fix i userReducer, no ala:  
`customerId: action.user['custom:customerId'].endsWith('/customer/None') ? "" : action.user['custom:customerId']`